template/Makefile

```diff
@@ -11,7 +11,7 @@ install:
 
 #### Development ####
 # start commands
-dev: update-box
+dev: update-box custom_ca
 	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} ANSIBLE_ARGS='--skip-tags "test"' vagrant up --provision
```

Comment:
```suggestion
dev: update-box custom_ca
	SSL_CERT_FILE=${SSL_CERT_FILE} CURL_CA_BUNDLE=${CURL_CA_BUNDLE} CUSTOM_CA=${CUSTOM_CA} ANSIBLE_ARGS='--skip-tags "test"' vagrant up --provision
```
Also need to add this like it is in `make up`
![image](https://user-images.githubusercontent.com/51820995/92929400-c2b5cc80-f440-11ea-98c3-cef0ca90db1b.png)

Reply:
Ahh ofc! Totally forgot that. Thanks for the reminder!